ros2_ws/src/rcdt_franka/nodes/close_gripper_node.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: Alliander N. V.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import rclpy
+from rclpy.node import Node
+from rclpy.action import ActionClient
+from rclpy.executors import MultiThreadedExecutor
+from rclpy.callback_groups import MutuallyExclusiveCallbackGroup
+from rclpy import logging
+from franka_msgs.action import Grasp
+from std_srvs.srv import Trigger
+
+ros_logger = logging.get_logger(__name__)
+
+
+class CloseGripper(Node):
+    def __init__(self) -> None:
+        super().__init__("close_gripper")
+
+        cbg_service = MutuallyExclusiveCallbackGroup()
+        self.create_service(
+            Trigger, "close_gripper", self.callback, callback_group=cbg_service
+        )
+
+        cbg_client = MutuallyExclusiveCallbackGroup()
+        self.client = ActionClient(
+            self, Grasp, "/fr3_gripper/grasp", callback_group=cbg_client
+        )
+
+    def callback(
+        self, _request: Trigger.Request, response: Trigger.Response
+    ) -> Trigger.Response:
+        response.success = self.close_gripper()
+        return response
+
+    def close_gripper(self) -> bool:
+        goal = Grasp.Goal()
```

Comment:
Dit `goal` is altijd hetzelfde, of niet? Dan is de definitie hier in `close_gripper()` niet optimaal